lib/internal/bootstrap/switches/does_not_own_process_state.js

```diff
@@ -9,6 +9,7 @@ process.abort = unavailable('process.abort()');
 process.chdir = unavailable('process.chdir()');
 process.umask = wrappedUmask;
 process.cwd = rawMethods.cwd;
+process.cwd.resetCwdCache = () => {};
```

Comment:
Can we use a symbol instead?

Reply:
I think we can simply hang it onto `internalBinding('process_methods')`. That way it's not visible to users at all.